recipe/meta.yaml

```diff
@@ -29,6 +29,8 @@ test:
   requires:
     - coverage
     - ipython
+    # seemingly needed for ipython_genutils
+    - jupyter
```

Comment:
no, you don't want to depend on `jupyter`: it hauls in `qt` and a lot of other stuff...

Reply:
This is just in testing.  If you have another fix, please do it instead.